optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -155,6 +162,49 @@ class AddBlurryImagePlaceholders {
     });
     return false;
   }
+
+  /**
+   * Checks if an image should have a blurred image placeholder.
+   * @param {Node} node The DOM element that is being checked to see if it
+   * should have a blurred placeholder.
+   * @param {string} src The image source that is being checked.
+   * @param {string} tagName The type of element that is being checked.
+   * @return {boolean} Whether or not the element should have a blurred
+   * placeholder child.
+   * @private
+   */
+  passesHueristic_(node, src, tagName) {
+    // Checks to see if the image is a jpeg.
```

Comment:
What's the rational for only blurring jpegs? Maybe add a quick explanation to the comment. Why not PNGs?

Reply:
+1. For some context, Addie and I discussed in person and after looking at popular AMP pages in the wild, using "responsive JPEG or video" seems like a good, conservative starting point and would be easy for publishers to understand.

Let's definitely document this in the function comment.

Reply:
Yup, the comment would be more interesting if it explained the why.